demos/top-app-bar.html

```diff
@@ -137,6 +201,36 @@
         drawer.open = true;
       });
 
+      var iconSection = document.getElementById('iconSection');
+      var shortCheckbox = document.getElementById('short');
+      var oneRightIcon = document.getElementById('no-right-icon');
+      var rtlCheckBox = document.getElementById('rtl-checkbox');
+
+      shortCheckbox.addEventListener('change', function() {
+        this.checked ? appBarEl.classList.add('mdc-top-app-bar--short') : appBarEl.classList.remove('mdc-top-app-bar--short');
+        this.checked ? appBarEl.classList.add('mdc-top-app-bar--short__right-icon') : appBarEl.classList.remove('mdc-top-app-bar--short__right-icon');
+
+        appBar.destroy();
+        appBar = mdc.topAppBar.MDCTopAppBar.attachTo(appBarEl);
+
+        if(!this.checked){
+          appBarEl.classList.remove('mdc-top-app-bar--short-collapsed');
+        }
+      });
+
+      oneRightIcon.addEventListener('change', function()  {
+        if(this.checked) {
+          iconSection.innerHTML = '';
```

Comment:
Can we apply a class to the icon to set its display to none, rather than tweaking innerHTML? This doesn't seem like how we'd want anyone to actually do things.

Reply:
Yes, but we'd have to change the `totalActionItems` adapter method to ignore `display:none`